apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Snippets/SnippetFixture.cs

```diff
@@ -0,0 +1,132 @@
+﻿// Copyright 2017 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#region
+
+using System;
+using System.Threading.Tasks;
+using Google.Cloud.Spanner.Admin.Database.V1;
+using Xunit;
+// ReSharper disable MemberCanBePrivate.Global
+
+#endregion
+
+namespace Google.Cloud.Spanner.Data.Snippets
+{
+    // ReSharper disable once ClassNeverInstantiated.Global
+    [CollectionDefinition(nameof(SnippetFixture))]
+    public class SnippetFixture : IDisposable, ICollectionFixture<SnippetFixture>
+    {
+        private readonly Lazy<Task> _creationTask;
+
+        public string TestInstanceName => "spannerinstance";
+
+        public string TestProjectName => Environment.GetEnvironmentVariable("TEST_PROJECT") ?? "cloud-sharp-jenkins";
```

Comment:
Actually I am lazy and since we don't have a getting started guide that says to set this -- this code will allow the tests to "just work" as long as your in cloudsharpeng